prototype-starter/app/filters.js

```diff
@@ -1,3 +1,3 @@
-const addFilter = require('govuk-prototype-kit').addFilter
+const addFilter = require('govuk-prototype-kit').nunjucks.addFilter
```

Comment:
What's the thinking behind having a `nunjucks` object like this?

Reply:
Also, I'm curious why you've written `const addFilter = require('govuk-prototype-kit').nunjucks.addFilter` rather than `const { addFilter } = require('govuk-prototype-kit').nunjucks`. Is there a reason to prefer one over the other? Do we need to be consistent about which one we use in the starter files?